data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2142 @@
+
+
+# Medium Warships
+
+ship "Diaspersi"
+	sprite "ship/avgi diaspersi"
+	thumbnail "thumbnail/avgi diaspersi"
+	swizzle 0
+	attributes
+		category "Medium Warship"
+		licenses
+			"Avgi Military"
+		"cost" 2400000
+		"shields" 2400
+		"hull" 12000
+		"required crew" 15
+		"bunks" 36
+		"mass" 270
+		"drag" 1.8
+		"heat dissipation" 1.8
+		"fuel capacity" 2400
+		"cargo space" 120
+		"outfit space" 420
+		"weapon capacity" 96
+		"engine capacity" 114
+		"anchor point" 2
+		weapon
+			"blast radius" 360
+			"shield damage" 2400
+			"hull damage" 1800
+			"hit force" 2400
+	outfits
+		"Phased Array Laser (Green)" 2
+		"Fleck Turret"
+		"Fleck Round" 2400
+		"Sidereal Launcher" 2
+		"Sidereal Missile" 84
+		"Equinox Torpedo Stockpile" 2
+		"Equinox Torpedo" 24
+		
+		"Luminous Fusion Core"
+		"Ultracapacitor Bank"
+		"Diffuse Deflector"
+		"Optical Lasing Generator"
+		"Multispectral Scanner" 2
+		"Concealment Scales"
+		"Deuterium Slush Tank"
+		"Electrostatic Ramscoop"
+		"Electromotive Rifle" 36
+		
+		"Z-600 Fusion Torch"
+		"XY-420 Radial Thrusters"
+		"Hyperdrive"
+
+	gun -18 -9 "Sidereal Launcher"
+		angle -90
+	gun 18 -9 "Sidereal Launcher"
+		angle 90
+	turret -9 12 "Phased Array Laser (Green)"
+	turret 9 12 "Phased Array Laser (Green)"
+	turret 0 51 "Fleck Turret"
+
+	bay "Fighter" 0 -63
+		over
+	bay "Fighter" -36 18
+		under
+	bay "Fighter" 36 18
+		under
+	bay "Drone" -21 -75
+	bay "Drone" 21 -75
+	bay "Drone" -21 -54
+	bay "Drone" 21 -54
+	bay "Drone" -24 -30
+	bay "Drone" 24 -30
+
+	engine 0 144
+	"reverse engine" -6 141
+		angle -35
+	"reverse engine" 6 141
+		angle 35
+	"steering engine" -9 -135
+		angle 90
+		right
+	"steering engine" 9 -135
+		angle -90
+		left
+	"steering engine" -6 141
+		angle 90
+		left
+	"steering engine" 6 141
+		angle -90
+		right
+	
+	leak "leak" 60 60
+	leak "flame" 60 90
+	leak "big leak" 90 30
+	explode "tiny explosion" 9
+	explode "small explosion" 21
+	explode "medium explosion" 21
+	explode "large explosion" 9
+	"final explode" "final explosion medium"
+
+	description "The Diaspersi-class Escort Carrier is a light carrier capable of providing carrier operations support to smaller fleets, while also being capable of independent strike missions with minimal support. Seen escorting large merchant convoys and small Guard fleets alike, the Diaspersi can support two wings of combat drones as well as a single wing of larger fighters."
+
+
+ship "Difraktos"
+	plural "Difraktos"
+	sprite "ship/avgi difraktos"
+	thumbnail "thumbnail/avgi difraktos"
+	swizzle 0
+	attributes
+		category "Medium Warship"
+		licenses
+			"Avgi Military"
+		"cost" 3600000
```

Comment:
```suggestion
		"cost" 5400000
```
Larger, tougher and better-armed than any human medium warship, so higher than the 3.86M for the Bastion, or the 4.2M for the Osprey.